### ISSUE TITLE
Unable to empty property value

### ISSUE THREAD

**Issue by zaplvlad** [payloadcms/payload#424]
Property value cannot be cleaned up.

## Steps to reproduce
1. Configure "global" collection with localized property as following.

```
import { GlobalConfig } from 'payload/types';

export const globalCollection: GlobalConfig = {
  slug: 'globalCollection',
  label: 'Global Collection',
  fields: [
    {
      name: 'someProperty',
      type: 'text',
      localized: true,
    },
  ],
};
```

2. Configure two locales for CMS.

`locales: ['en', 'da-DK']`

3. Go to CMS and fill up this property with some data for any locale.
4. Save document.

![1](https://user-images.githubusercontent.com/43339104/150782822-8fc9fb36-055f-43c7-a346-45966725f00d.png)

5. Reload page to make sure property value was saved.
6. Cleanup property value.
7. Save document.

![2](https://user-images.githubusercontent.com/43339104/150782883-8eba52dc-b7fd-4172-ac71-087b4761eacd.png)

8. Reload page again.

## Current Behavior
You will see that property data wasn't removed and still saved. It is possible to change value but not cleanup. The same behavior present in other property types, not only for "text".

## Expected result
It should be possible to empty property data.

**Comment by jmikrut** [comment]
Hey @zaplvlad — good catch!

We can fix this pretty swiftly. Keep an eye out. We'll update here with a fix coming up soon.

Thanks for the report!

**Comment by jmikrut** [comment]
Hey @zaplvlad — this is now fixed and released as beta in `payload@0.14.20-beta.0`.

Thank you!!
